e2e/test/lb-with-udp-ports-mode/chainsaw-test.yaml

```diff
@@ -0,0 +1,66 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/kyverno/chainsaw/main/.schemas/json/test-chainsaw-v1alpha1.json
+apiVersion: chainsaw.kyverno.io/v1alpha1
+kind: Test
+metadata:
+  name: lb-with-udp-ports-mode
+  labels:
+    all:
+    lke:
+spec:
+  namespace: "lb-with-udp-ports-mode"
+  steps:
+    - name: Create pods and services
+      try:
+        - apply:
+            file: create-pods-services.yaml
+      catch:
+        - describe:
+            apiVersion: v1
+            kind: Pod
+        - describe:
+            apiVersion: v1
+            kind: Service
+    - name: Check that loadbalancer ip is assigned
+      try:
+      - assert:
+          resource:
+            apiVersion: v1
+            kind: Service
+            metadata:
+              name: svc-test
+            status:
+              (loadBalancer.ingress[0].ip != null): true
+    - name: Fetch nodebalancer config for port 7070
+      try:
+        - script:
+            content: |
+              set -e
+              
+              echo "Starting e2e test"
+
+              nbid=$(KUBECONFIG=$KUBECONFIG NAMESPACE=$NAMESPACE LINODE_TOKEN=$LINODE_TOKEN ../scripts/get-nb-id.sh)
+
+              echo "Nodebalancer ID: $nbid"
+
+              nbconfig=$(LINODE_TOKEN=$LINODE_TOKEN NBID=$nbid ../scripts/get-nb-config.sh)
+              config_id=$(echo $nbconfig | jq -r '.id')
+
+              node_ip=$(curl -k \
+                --url $LINODE_URL/v4beta/nodebalancers/$nbid/configs/$config_id/nodes \
+                -H 'accept: application/json' \
+                -H "Authorization: Bearer $LINODE_TOKEN" | jq -r '.data[0].address')
+
+              echo "Nodebalancer config found, adding a node"
+              curl -sk --request POST \
```

Comment:
We don't need to add node as it should already be there as we had created deployment and service earlier. There should be one node already in the nodebalancer config. Are we testing something specific here by adding new node ourselves?